### PR TITLE
Adds missing data: event, riskyUser, riskDetection

### LINKED ISSUE #610: Missing Data: {event-id}
Test Cases:
- [ ] event-get-attachments-v1-csharp-V1-executes
- [ ] get-event-csharp-V1-executes
- [ ] get-event-multiple-locations-csharp-V1-executes
- [ ] get-instances-csharp-V1-executes

### LINKED ISSUE #793: Missing Data: {riskyUser-id}
Test cases:
- [ ] get-riskyuserhistoryitem-2-csharp-V1-executes
- [ ] get-riskyuser-1-csharp-V1-executes
- [ ] get-riskyuserhistoryitem-1-csharp-V1-executes

### LINKED ISSUE #794: Missing Data: {riskDetection-id}
Test case:
- [ ] get-riskdetection-1-csharp-V1-executes

### PR DESCRIPTION
**Pull request by zengin**
- fixes #610 
- fixes #794 
- fixes #793 
- changes root cause for #792 
- changes root cause for #617
- changes root cause for #779
- changes root cause for #651

### CODE REVIEW

**Comment by zengin** [comment]
PowerShell tests are cancelled, opened https://github.com/microsoftgraph/msgraph-sdk-raptor/issues/802 to disable them when the changes are unrelated.

**Review by fey101 — APPROVED** [review]

**Comment by fey101** [comment]
Great progress thanks to this PR. Thanks.

**Comment by fey101** [comment]
@zengin , I don't see changes related to changes root cause for Missing Data: {externalConnectors.externalConnection-id} #617. Did you miss out on an update or am I missing out on sth?

File: TestsCommon/CSharpKnownIssues.cs [hunk 1]
```diff
@@ -658,7 +643,7 @@ public static class CSharpKnownIssues
                 { "callrecord-getdirectroutingcalls-csharp-V1-executes", NoProgrammaticWayKnownIssue with { GitHubIssue = "https://github.com/microsoftgraph/msgraph-sdk-raptor/issues/320" } },
                 { "get-connectionoperation-csharp-V1-executes", EphemeralOperationDataKnownIssue with { GitHubIssue = "https://github.com/microsoftgraph/msgraph-sdk-raptor/issues/692"} },
                 { "get-externalgroup-csharp-V1-executes", MissingPermissionScopeKnownIssue with { GitHubIssue = "https://github.com/microsoftgraph/msgraph-sdk-raptor/issues/693" } },
-                { "get-schema-csharp-V1-executes", MissingDataKnownIssue with { GitHubIssue = "https://github.com/microsoftgraph/msgraph-sdk-raptor/issues/617" } },
```
Comment (zengin) [inline TestsCommon/CSharpKnownIssues.cs:661]: @fey101 #617 is not resolved, I only marked the blocker that needs to be resolved before the missing data.

Comment (fey101) [reply TestsCommon/CSharpKnownIssues.cs:661]: And the blocker is the 'get-schema-csharp-V1-executes'? Am looking to solve issue 617 as a missing data issue but I am not clear on its status.

**Pull request merged**